CHANGELOG-3.3.md

```diff
@@ -80,6 +80,13 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 
 - Compile with [*Go 1.12.9*](https://golang.org/doc/devel/release.html#go1.12) including [*Go 1.12.8*](https://groups.google.com/d/msg/golang-announce/65QixT3tcmg/DrFiG6vvCwAJ) security fixes.
 
+### Breaking Changes
+
+- Improve [client balancer failover against multiple endpoints](https://github.com/etcd-io/etcd/pull/11184).
+  - Fix ["kube-apiserver: failover on multi-member etcd cluster fails certificate check on DNS mismatch" (kubernetes#83028)](https://github.com/kubernetes/kubernetes/issues/83028).
+- Improve [client balancer failover against IPv6 endpoints](https://github.com/etcd-io/etcd/pull/11211).
```

Comment:
IIUC, the fix is for parsing IPv6 in etcd client, not necessarily related to client balancer failover. So maybe something like `Fix IPv6 endpoint parsing in client`?

Given that they are bug fixes, maybe put these under section `client v3` instead of `breaking changes`?

Reply:
@jingyih sure, sounds good. Thanks!!